src/components/ProductBlock/index.tsx

```
@@ -0,0 +1,83 @@
+export const CardHeading = ({ eyebrow, icon, title, light }: CardHeadingProps) => {
+    return (
+      <div className={clsx(styles.cardHeader, light && styles.light)}>
+        <div className={clsx(styles.iconWrapper, light && styles.light)}>
+            <Icon icon={icon} color={light ? "var(--surface-brand-default)" : "#fff"} size="32px" />
+        </div>
+        <div className={styles.cardHeaderContent}>
+            {eyebrow && <h5 className={styles.eyebrow}>{eyebrow}</h5>}
+            <Heading as="h4" className={styles.h4}>
+              <div
+                className={styles.h4}
+                dangerouslySetInnerHTML={{ __html: title }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)